test/pummel/test-watch-file.js

```diff
@@ -21,31 +21,26 @@
 
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const fixtures = require('../common/fixtures');
 
 const f = fixtures.path('x.txt');
 
-let changes = 0;
+const callback = common.mustCall((curr, prev) => {
```

Comment:
```suggestion
const callback = common.mustCallAtLeast((curr, prev) => {
```
To keep the old behaviour.